src/single-spa-vue.js

```diff
@@ -88,14 +88,17 @@ function mount(opts, mountedInstances, props) {
           }
         } else {
           domEl = appOptions.el;
+          if (!domEl.parentNode) {
+            throw Error(
+              `If appOptions.el is provided to single-spa-vue, the dom element must exist in the dom. Was provided as ${appOptions.el.outerHTML}`
+            );
+          }
           if (!domEl.id) {
             domEl.id = `single-spa-application:${props.name}`;
           }
-          appOptions.el = `#${CSS.escape(domEl.id)}`;
         }
       } else {
         const htmlId = `single-spa-application:${props.name}`;
-        appOptions.el = `#${CSS.escape(htmlId)}`;
```

Comment:
If appOptions.el isn't provided to single-spa-vue, don't we need to add it so that Vue knows where to mount?

Reply:
Actually, `appOptions.el` is defined a few lines below: https://github.com/single-spa/single-spa-vue/pull/106/files#diff-d3b89dd088913971f21ae339bd0ea76769602c03acb5d9e1efc116efe355bb28R125

The fact is from line 82 to 106 we are creating the `domEl`.
But, we cannot assign it to `appOptions.el` before handling the "replaceMode" behaviour.